src/core/Alert/index.stories.tsx

```diff
@@ -0,0 +1,124 @@
+import styled from "@emotion/styled";
+import { Snackbar } from "@material-ui/core";
+import CheckCircleOutlineIcon from "@material-ui/icons/CheckCircleOutline";
+import ErrorOutlineIcon from "@material-ui/icons/ErrorOutline";
+import { storiesOf } from "@storybook/react";
+import React from "react";
+import Button from "../Button";
```

Comment:
@tihuan Am I importing our Button incorrectly? I'm getting a lot of these errors (but not when I use the base material-ui button)

![image](https://user-images.githubusercontent.com/53838890/110863978-53e1f080-8276-11eb-947e-ddec36f7bf31.png)